llvm/docs/LangRef.rst

```diff
@@ -3030,8 +3030,7 @@ as follows:
 ``S<size>``
     Specifies the natural alignment of the stack in bits. Alignment
     promotion of stack variables is limited to the natural stack
-    alignment to avoid dynamic stack realignment. The stack alignment
-    must be a multiple of 8-bits. If omitted, the natural stack
```

Comment:
Please do not delete the `8-bits`!!!

Reply:
It is implied in the added paragraph.

Reply:
This is an 8-bit per byte world and your downstream target differs from that assumption. Don't play games.

Reply:
Please be constructive in your criticism.
I'm marking this conversation as resolved unless you have any specific concerns.

Reply:
Not resolved. You substitute 8-bit by byte to remove the 8-bit restriction.

Reply:
constructive criticism?

Reply:
cheap trick

Reply:
I think it's fine omit the 8-bit mention here. It's only mentioned once for `S` and not any of the other alignment specifications. I looked through LangRef a bit, and with the notable exception of this sentence, the wording always tries to be byte-size agnostic.

Reply:
LangRef probably *should* mention somewhere that bytes are 8 bits large in LLVM -- this just isn't the right place, as it's a global assumption, not specific to only this one thing.

Not sure it makes sense to actually add that note *now* though, if there are plans to lift this restriction in the near future.